src/index.json

```diff
@@ -984,6 +984,39 @@
                     "version": "0.1.2"
                 }
             }
+        ],
+        "aladdin": [
```

Comment:
Change this to 'find', or something similar, as this is the name a user will see. 'aladdin' will not give them information on what the extension will provide.